sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1126,6 +1133,44 @@ Exceptions:
 
 |===
 
+==== New Event Member Functions
+
+Table {counter: tableNumber}. Additional member functions of the `sycl::event` class.
+[cols="2a,a"]
+|===
+|Member function|Description
+
+|
+[source,c++]
+----
+template <typename Param>
+typename Param::return_type 
+event::ext_oneapi_get_profiling_info(node Node) const;
+----
+
+| Queries the profiling information of a SYCL Graph node for the graph
+  execution associated with this SYCL event. If the requested info is
+  not available when this member function is called due to incompletion of
+  command groups associated with the event, then the call to this member
```

Comment:
```suggestion
  command groups associated with the node, then the call to this member
```
My understanding is that the block would be related to the graph completion. Is that intended or do we want to restrict it to node only?